init.lua

```diff
@@ -116,6 +116,15 @@ function sharedtags.new(def)
     return tags
 end
 
+function getidx(tag)
+    if tag.sharedtagindex then
+        -- Add arbitrarily large number to always be at the end
+        return tag.sharedtagindex + 0
```

Comment:
I don't get this. How is zero large?

Reply:
Sorry, I forgot to change this back, I was just testing to see if an offset would affect normal users. This should be a number like 10, so that the shared tags always appear after normal tags.